packages/e2e-tests/src/plugins/web-stories-cors-error.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Plugin Name: E2E Tests CORS error
+ * Plugin URI:  https://github.com/googleforcreators/web-stories-wp
+ * Description: CORS errors on purpose for testing.
+ * Author:      Google
+ * Author URI:  https://opensource.google.com
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
```

Comment:
```suggestion
 * @copyright 2022 Google LLC
 * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
```

We should fix this for the other e2e test plugins as well. Can be done in this PR too.